etc/di.xml

```diff
@@ -267,4 +267,13 @@
     <type name="Amasty\Promo\Plugin\Quote\Model\Quote\TotalsCollectorPlugin">
         <plugin name="TotalsCollectorPlugin" type="Bolt\Boltpay\Plugin\Amasty\Promo\Quote\Model\Quote\TotalsCollectorPlugin"/>
     </type>
+    <type name="Magento\SalesRule\Model\Utility">
+        <plugin name="Bolt_Boltpay_SalesRule_Model_Utility_Plugin"
```

Comment:
@dumega  Why do you add this plugin? I think we have it in this file https://github.com/BoltApp/bolt-magento2/blob/master/etc/di.xml#L212-L216 We should delete it, right?

Reply:
@johnmccombs1 I have removed it.